src/utils/clmfates_interfaceMod.F90

```diff
@@ -2593,6 +2643,116 @@ end subroutine wrap_canopy_radiation
 
  ! ======================================================================================
 
+ subroutine WrapSeedGlobal(this,is_restart_flag)
```

Comment:
Can we rename `WrapSeedGlobal` and `wrap_seed_dispersal` to be more informative?

As far as I understand `wrap_seed_dispersal` gives seed to each gridcell/site (and resets outgoing seeds) from the gathered data and `WrapSeedGlobal` actually disperses the seeds? 

I would suggest something like `update_site_seed_inout` for `wrap_seed_dispersal` and then `WrapSeedDispersalGlobal` for `WrapSeedGlobal`